app/recipe/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+class Recipe(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.CharField(max_length=255)
```

Comment:
You could use a `Textfield` for this to avoid the 255 character limitation. Actually if you used it for every string then you do not need to limit to 255 chars